### DISCUSSION TITLE
请问,基于 Key 集群限流,是否支持动态添加key?

### DISCUSSION THREAD

**Discussion by MrOldPeng** [higress-group/higress#1275] [Q&A]
实现业务层的针对token限流

**Comment by hanxiantao** [comment]
> 实现业务层的针对token限流

现在是只能支持在consul上修改wasm插件对应的配置（实际上是修改Kubernetes中WasmPlugin对应的CRD）

因为wasm插件加载配置只会执行一次，也没办法搞一个job一直调用外部接口动态调整策略

如果业务这边系统需要动态的修改，可以外部调用consul的修改配置接口实现
